src/main/java/sirius/biz/jobs/batch/file/FileExportJob.java

```diff
@@ -151,21 +151,20 @@ private boolean shouldUseProvidedOutputDirectory() {
     /**
      * Determines the effective file name to use.
      *
-     * @param suffix an optional suffix to append to the name to generate a unique file name
+     * @param suffix      an optional suffix to append to the name to generate a unique file name
+     * @param includeDate whether the current date should be appended to the filename
      * @return a full file name including the file extension
      */
-    protected String determineEffectiveFilename(String suffix) {
```

Comment:
Keep and deprecate old signature, forwarding to the new one in compatible fashion?